packages/split-view/stories/split-view.stories.ts

```diff
@@ -0,0 +1,272 @@
+                    readable English. Many desktop publishing packages and web
+                    page editors now use Lorem Ipsum as their default model
+                    text, and a search for 'lorem ipsum' will uncover many web
+                    sites still in their infancy. Various versions have evolved
+                    over the years, sometimes by accident, sometimes on purpose
+                    (injected humour and the like).
+                </p>
+            </div>
+        </sp-split-view>
+    `;
+};
+
+export const MultipleLevels = (): TemplateResult => {
+    return html`
+        <sp-split-view
+            resizable
+            primary-min="50"
+            primary-max="200"
+            secondary-min="50"
+            style="height: 400px; width: 600px;"
+        >
+            <div>
+                <h1>First panel - Level 1</h1>
+                <p>
+                    Lorem Ipsum is simply dummy text of the printing and
+                    typesetting industry. Lorem Ipsum has been the industry's
+                    standard dummy text ever since the 1500s, when an unknown
+                    printer took a galley of type and scrambled it to make a
+                    type specimen book. It has survived not only five centuries,
+                    but also the leap into electronic typesetting, remaining
+                    essentially unchanged. It was popularised in the 1960s with
+                    the release of Letraset sheets containing Lorem Ipsum
+                    passages, and more recently with desktop publishing software
+                    like Aldus PageMaker including versions of Lorem Ipsum.
+                </p>
+            </div>
+            <div>
+                <h2>Second panel - Level 1</h2>
+                <sp-split-view
+                    vertical
+                    resizable
+                    primary-min="50"
+                    primary-size="100"
+                    secondary-min="50"
+                    style="height: 300px;"
+                >
+                    <div>
+                        <h3>First panel - Level 2</h3>
+                        <p>
+                            Lorem Ipsum is simply dummy text of the printing and
+                            typesetting industry.
+                        </p>
+                    </div>
+                    <div>
+                        <h4>Second panel - Level 2</h4>
+                        <p>
+                            It is a long established fact that a reader will be
+                            distracted by the readable content of a page when
+                            looking at its layout.
+                        </p>
+                    </div>
+                </sp-split-view>
+            </div>
+        </sp-split-view>
+    `;
+};
+
+export const OnePaneNoSplitter = (): TemplateResult => {
+    return html`
+        <sp-split-view style="height: 200px"></sp-split-view>
```

Comment:
Is this intended? Seems like this should have a single `div` (or similar) with some content in it based on that name?

Reply:
There was indeed a inner <div> missing, I fixed this.